File: api/subs.py
import os
import requests
from http.server import BaseHTTPRequestHandler

class handler(BaseHTTPRequestHandler):
    def do_GET(self):
        channel_id = "YOUR_CHANNEL_ID"  # Replace with your actual channel ID
        api_key = os.getenv("YOUTUBE_API_KEY")

        url = f"https://www.googleapis.com/youtube/v3/channels?part=statistics&id={channel_id}&key={api_key}"
        response = requests.get(url)
        data = response.json()

        count = data["items"][0]["statistics"]["subscriberCount"]
        self.send_response(200)
        self.send_header("Content-type", "text/plain")
        self.end_headers()
        self.wfile.write(count.encode())

File: api/subs.js
// api/subs.js
export default async function handler(req, res) {
  try {
    const response = await fetch(
      `https://www.googleapis.com/youtube/v3/channels?part=statistics&id=UC0KdIoPfAh_aKEOcBlexMMw&key=${process.env.YT_API_KEY}`
    );
    const data = await response.json();

    if (!data.items || data.items.length === 0) {
      throw new Error("Channel not found or API error");
    }

    const subs = data.items[0].statistics.subscriberCount;
    res.status(200).json({ subscribers: subs });
  } catch (error) {
    console.error(error);
    res.status(500).json({ subscribers: "Unknown" });
  }
}
